test/ibuf.c

```diff
@@ -34,17 +34,26 @@ ibuf_basic()
 
 	fail_unless(ibuf_used(&ibuf) == 0);
 
+	ptr = ibuf_alloc(&ibuf, UINT32_MAX);
```

Comment:
The test will fail if the system has less than 4 GB of RAM. People [complain](https://github.com/tarantool/tarantool/issues/6718).

Reply:
If we change unsigned to uint64_t and __builtin_clz to __builtin_clzll, there will be no need to test it.